### DISCUSSION TITLE
Is there an 2MFA login example and login to api endpoind?

### DISCUSSION THREAD

**Discussion by biapar** [directus/directus#5261] [Archive]
Is there an 2MFA login example and login to api endpoind?

**Comment by rijkvanzanten** [comment]
Yes, see https://docs.directus.io/reference/api/system/authentication/#login

**Reply by biapar** [reply]
I'll check. I've to manage some sensitive data. Which is the security level?
Gdpr compliant?

**Reply by rijkvanzanten** [reply]
Directus will only store what you feed it. GDPR compliance fully depends on how you end up using Directus 👍🏻
